src/main/java/chess/Board.java

```diff
@@ -1,81 +1,151 @@
 package chess;
 
-import pieces.Pawn;
-
+import pieces.Piece;
+import static utils.StringUtils.*;
 import java.util.ArrayList;
 import java.util.List;
-
-
 public class Board {
 
-    private List<Pawn> whitePawnList = new ArrayList<>();
-    private List<Pawn> blackPawnList = new ArrayList<>();
+    private List<Piece> whitePieceList = new ArrayList<>();
+    private List<Piece> blackPieceList = new ArrayList<>();
 
-    private String startLine = "* * * * * * * *";
+    private List<Piece> whitePawnList = new ArrayList<>();
+    private List<Piece> blackPawnList = new ArrayList<>();
 
-    private final int PAWNCOUNT = 8;
+    private String startLine = "********";
 
-    public Pawn findPawn(Pawn.Color color, int index) {
-        if (color == Pawn.Color.WHITE) {
+    private final int PAWN_COUNT = 8;
```

Comment:
인스턴스 변수에 맞지 않은 네이밍 컨벤션을 사용하셨네요. 

```suggestion
    private final int pawnCount = 8;